sp_mocks/imsharer/frontend/src/App.vue

```diff
@@ -6,7 +6,7 @@ import emitter from '@/plugins/mitt';
 
 onMounted(async () => {
   await layer8.initEncryptedTunnel({
-    providers: ["http://localhost:6001"],
+    providers: ["http://localhost:8000"],
```

Comment:
We should leave this as 6001 along with the others and register it as a new client instead

Reply:
sure, i change it so it follow the manual test case, we can update the test case then